@Test public void should_include_message_built_with_given_throwable_name(){
  assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> shouldHaveThrown(OutOfMemoryError.class)).withMessage("OutOfMemoryError should have been thrown");
}
